### PR TITLE
Add new section to About page https://github.com/18F/eregs-platform/issues/57

### PR DESCRIPTION
**Pull request by tadhg-ohiggins**
Use “Stop Comparing” /
Now that we have a section /
Clarifying it.

### CODE REVIEW

File: atf_eregs/static/regulations/css/less/module/custom/about.less [hunk 1]
```diff
@@ -24,6 +24,38 @@
...
+    margin-left: 20px;
+}
+
+.xabout-timeline-red-icon {
+    content: "\e642";
+    color: #071635;
+    margin-left: -32px;
+    color: green;
+    border: 10px solid red;
+}
```
Comment (cmc333333) [inline atf_eregs/static/regulations/css/less/module/custom/about.less]: Should this rule be removed? The class name includes an x at the beginning.

File: atf_eregs/static/regulations/css/less/module/custom/about.less [hunk 2]
```diff
@@ -24,6 +24,38 @@
...
+    font-weight: 400;
+    font-variant: normal;
+    text-transform: none;
+    line-height: 1;
+    -webkit-font-smoothing: antialiased;
+    content: "\e642";
+    xcolor: #071635;
+    color: #d14124;
+    font-size: 18px;
+}
```
Comment (cmc333333) [inline atf_eregs/static/regulations/css/less/module/custom/about.less]: This is a bit gnarly; what about using this instead?

```html
<span class="cf-icon cf-icon-delete-round">
```

File: atf_eregs/templates/regulations/about.html [hunk 3]
```diff
@@ -25,3 +25,33 @@
...
+    </div>
+
+    <div class="content-primary column-r">
+        <p class="about-timeline">The <strong>regulation timeline</strong> shows recent revisions of the regulation organized by latest effective date.</p>
+
+        <ol class="about-ol bump">
+            <li>The <strong>date picker</strong> allows users to find what revision was effective on a specific date.</li>
+            <li><strong>Federal Register notice(s)</strong> associated with a revision are presented and organized by publication date. These links will
+            take you to the notice on the Office of the Federal Register’s website.</li>
+            <li><strong>Compare</strong> any two revisions of the regulation, word for word, by selecting a date in the menu and clicking “Show Differences”.</li>
```
Comment (cmc333333) [inline atf_eregs/templates/regulations/about.html]: Solid

File: atf_eregs/static/regulations/css/less/module/custom/about.less (hunk 1 above)
Comment (tadhg-ohiggins) [inline atf_eregs/static/regulations/css/less/module/custom/about.less]: Yup, removed.

File: atf_eregs/static/regulations/css/less/module/custom/about.less (hunk 2 above)
Comment (tadhg-ohiggins) [inline atf_eregs/static/regulations/css/less/module/custom/about.less]: So much cleaner, thanks!

File: atf_eregs/static/regulations/css/scss/module/custom/_about.scss [hunk 4]
```diff
@@ -24,6 +24,19 @@
     background: $atf_carrots;
 }
 
+.about-stopcompare {
+    margin-top: 80px;
+    margin-left: 20px;
+}
+
+.icon-red {
+    color: #d14124;
```
Comment (cmc333333) [inline atf_eregs/static/regulations/css/scss/module/custom/_about.scss]: It's a shame we're using a hard-coded color in -site here, but thanks for highlighting that. Created https://github.com/eregs/regulations-site/issues/463

File: atf_eregs/static/regulations/css/scss/module/custom/_about.scss [hunk 5]
```diff
@@ -24,6 +24,19 @@
...
+}
+
+img.about-stopcompare-img {
```
Comment (cmc333333) [inline atf_eregs/static/regulations/css/scss/module/custom/_about.scss]: NBD, but per https://pages.18f.gov/frontend/#css-specificity, this shouldn't include the `img`

Comment (tadhg-ohiggins) [inline atf_eregs/static/regulations/css/scss/module/custom/_about.scss]: Good point; addressed here https://github.com/18F/atf-eregs/pull/445

**Pull request merged**